<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(162, 255, 255, 255), stop:1 rgba(237, 223, 255, 255));
}

#widget, #widget_2 {
	background-color: rgba(255, 255, 255, 150);
border-radius: 10;
}

*{
	
	font: 12pt &quot;Stencil&quot;;
}

.QPushButton {
border: null;
	background-color: rgba(237, 223, 255, 255);
	border-radius: 10;
}

.QPushButton:hover {
border: null;
	background-color: rgba(191, 153, 235, 150);
	border-radius: 10;
}

.QTimeEdit {
border: null;
background: transparent;
border-bottom: 1px solid black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Установить будильник</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="alarm_edit">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="displayFormat">
          <string>H:mm:ss</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="set_alarm_btn">
         <property name="text">
          <string>Установить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="alam_status_label">
         <property name="text">
          <string>Будильник не установлен</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="notifications_label">
      <property name="text">
       <string>Уведломления

</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="trayCheckBox">
      <property name="text">
       <string>Скрывать в стрейт</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
